src/carousel/carousel.scss

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../text/index';
+@import '../button/button';
+
+@mixin iui-carousel {
+  @include iui-reset;
+  display: flex;
```

Comment:
Does flex do here anything?

Reply:
I thought it was necessary for `flex-direction: column;` to work, but it would appear it is not.  Removed.

Reply:
So now instead, you have changed HTML layout and putting nav first and then slides. For that, you are using flex and column-reverse direction. I am never a fan of inverting the html order unless there is a reason.
So what was the reason to do that?

Reply:
[w3 documentation](https://www.w3.org/TR/wai-aria-practices-1.2/#keyboard-interaction-4) says:
> If present, the rotation control is the first element in the Tab sequence inside the carousel. It is essential that it precede the rotating content so it can be easily located.
By reordering the HTML structure, if you hit tab it will target the back arrow, pause, dots, right arrow before it targets anything within the slides themselves.

Reply:
Thats nice, but.. Rotation control is not the same as Slide picker controls.
![image](https://user-images.githubusercontent.com/9351976/150853720-ee13e3ca-dddf-4945-a66e-788a777e35e6.png)

So in our case, this should be only pause/start button, which is fully custom and provided by user.

Reply:
Thats nice, but.. Rotation control is not the same as Slide picker controls.
![image](https://user-images.githubusercontent.com/9351976/150853720-ee13e3ca-dddf-4945-a66e-788a777e35e6.png)

So in our case, this should be only pause/start button, which is fully custom and provided by user.

Reply:
I believe this should be the order:
1. rotation control ("play"/"pause")
2. slide controls ("prev"/"next"/"dots")
3. slides (i.e. focusable elements inside the slides)

Reply:
Well, we do not provide special case for play/pause button. That one is user provided, if needed. I guess we could have documentation about it (that would be really nice).
Even though w3 does not say anything about controls and order, I guess from this point of view it might make sense to have those first. I will resolve this.

Reply:
Latest push now targets play/pause first.